### ISSUE TITLE
My friend can't play in my ogar server.

### ISSUE THREAD

**Issue by Tomi1311** [OgarProject/Ogar#469]
Hello, I created a ogar server and i can play in this but my friend not. We are using hamachi, when i give to him the ip. He open the console of google chrome and put in console this: connect("ws://25.98.96.180:1234") but he can't join the server. The error is this: WebSocket connection to connect (Here goes the ip) failed: Error in connection establishment: net::ERR_CONNECTION_TIMED_OUT. 
Sorry if my english is bad. I'm learning the language.

**Comment by LeninYT** [comment]
try make your port something else than 1234 like 4545 or simply 443

**Comment by Agarian** [comment]
that will not change anything.

**Comment by Agarian** [comment]
i tried that already.

**Comment by Tomi1311** [comment]
Hello, now I can't join. Any solution?

**Comment by Medusalix** [comment]
You could try http://agar.io?ip=

**Comment by Andrews54757** [comment]
no. it is a hamachi problem. i have seen too much of hamachi problems. solution: dont use hamachi just port foward and if you cannot, too bad

**Comment by ItsMeRomian** [comment]
Im with @Andrews54757, hamachi is just a pain in the ass

**Comment by NatsuTheGreat** [comment]
@vram4 Close?

**Comment by Agarian** [comment]
try using c9.io

**Comment by jollesnaas** [comment]
You can rent a VPS (Windows or Linux) of me: 4 euro/month/512MB RAM or 8 euro/month/1GB RAM, 20 euro/Year/512MB RAM or 40 euro/year/1GB RAM.

If you want to know more or order, please contact me on skype: jolle.snaas
*Free test included! 

With a vps everyone can join your server!

**Comment by Andrews54757** [comment]
i dont trust this . dont fall for it others! Only give money to trusted people
